pyramid_zipkin/request_helper.py

```diff
@@ -71,7 +71,12 @@ def should_not_sample_path(request):
     """
     blacklisted_paths = request.registry.settings.get(
         'zipkin.blacklisted_paths', [])
-    regexes = [re.compile(r) for r in blacklisted_paths]
+    # Only compile strings, since even recompiling existing
+    # compiled regexes takes time.
+    regexes = [
+        re.compile(r) if isinstance(r, str) else r
```

Comment:
basestring won't work for py3, which we support. To get this working nicely without a whole mess of if/else statements we could probably dep [six](https://pypi.python.org/pypi/six).

I thought this option of just making users `re.compile()` their unicode strings was the easiest. No one uses unicode in paths anyways.